app/components/request_form/request_form.html.erb

```diff
@@ -14,6 +14,20 @@
         <% end %>
 
         <%= c 'field', object: @request, attr: :text do |field| %>
+          <div
```

Comment:
Is there a specific reason for using single quotes in the HTML?

Reply:
What do you mean by this? If I look at the `RequestForm` component, there is actually a mixture of double and single quotes. Just briefly glancing this is the case with most of our HTML. We don't enforce any style guide so this is probably what has led to the inconsistency.

We do enforce single quotes with `rubocop` in ruby land, so it makes sense, I think.

Reply:
The way I understood it was double quotes for attributes in plain HTML and single quotes for "ruby enhanced HTML".